content/overview/overview/using-cloudgov-paas.md

```diff
@@ -3,7 +3,7 @@ menu:
   overview:
     parent: overview
 title: What you can do with the cloud.gov platform
-weight: 20
+weight: 13
```

Comment:
This order change is to help people get an idea of what cloud.gov provides *before* we start listing restrictions on who can use it and what tech is supported.